playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/volcano_tfbenchmarks.yaml

```diff
@@ -71,7 +71,7 @@
         set -o pipefail
         set -ex
         mkdir -p src/volcano.sh/
-        git clone https://github.com/volcano-sh/volcano src/volcano.sh/volcano
+        git clone https://github.com/volcano-sh/volcano src/volcano.sh/volcano -b release-0.1
```

Comment:
Consider the extensibility, use a variable to set the version of volcano?

Reply:
sure, will update